PageObjects/UI/CheckoutPage.js

```diff
@@ -0,0 +1,188 @@
+    this.expirationMonthSelect = page.locator("[aria-label='11']");
+    this.expirationYearSelect = page.locator("[aria-label='2030']");
+    this.comfirmationMessageUpdate = page.locator(
+      "div[role='alert]:has-text('Billing information updated!')"
+    );
+    this.shippingInfoH3 = page.locator("h3:has-text('Shipping information')");
+    this.firstNameValue = generalMethods.randomFirstName();
+    this.lastNameValue = generalMethods.randomLastName();
+    this.emailValue = generalMethods.randomEmail;
+    this.phoneNumberValue = `+${generalMethods.randomCardNumber}`;
+    this.adressValue = "Brace Ribnikar 12";
+    this.cityValue = "Novi Sad";
+    this.postalCodeValue = "21000";
+    this.countryValue = "Serbia";
+    this.cardholderValue = generalMethods.randomFirstName();
+    this.cardNumberValue = generalMethods.randomCardNumber;
+    this.cvvValue = "123";
+    this.cardTypeValue = "MasterCard";
+    this.checkFirsName = page.locator("span:text-is('First Name') + span");
+    this.checkLastName = page.locator("span:text-is('Last Name') + span");
+    this.checkEmail = page.locator("span:text-is('Email') + span");
+    this.checkPhone = page.locator("span:text-is('Phone') + span");
+    this.checkAdress = page.locator("span:text-is('Street and Number') + span");
+    this.checkCity = page.locator("span:text-is('City') + span");
+    this.checkPostalCode = page.locator("span:text-is('Postal Code') + span");
+    this.checkCountry = page.locator("span:text-is('Country') + span");
+    this.checkCardholder = page.locator("span:text-is('Cardholder') + span");
+    this.checkCardNumber = page.locator("span:text-is('Card Number') + span");
+    this.checkCvv = page.locator("span:text-is('CVV') + span");
+    this.checkCardType = page.locator("span:text-is('Card Type') + span");
+    this.checkExpDate = page.locator("span:text-is('Expiration Date') + span");
+    this.expDateValue = "11/30";
+    this.confirmButton = page.locator(
+      ".p-button:has-text('Place your order!')"
+    );
+  }
+
+  async rewiewItems() {
+    await expect(this.checkMessageFreeOfCharge).toBeVisible();
+    await expect(this.notWithMoney).toBeVisible();
+    await expect(this.productExist).toBeVisible();
+    const responsePromise = this.page.waitForResponse(
+      `${process.env.BASE_URL_API}${endpoints.customersApiEndpoint}/${process.env.USER_ID}/shipping-info`
+    );
+    await this.nextStepButton.click();
+    const response = await responsePromise;
+    await expect(response.ok()).toBeTruthy();
+    await expect(this.checkMessageCallMeDevil).toBeVisible();
+  }
+
+  async addCustomerInfo({
+    firstName = this.firstNameValue,
+    lastName = this.lastNameValue,
+    email = this.emailValue,
+    adress = this.adressValue,
+    phoneNumber = this.phoneNumberValue,
+    city = this.cityValue,
+    postalCode = this.postalCodeValue,
+    country = this.countryValue,
+  }) {
+    await this.page.setViewportSize({ width: 1600, height: 1200 });
```

Comment:
nije dobra opcija da ti funcija setuje viewport, to treba da bude na nivou testa